tsdb/operations/default.json

```diff
@@ -58,7 +58,11 @@
 {
   "name": "date-histo-entire-range-1d",
   "operation-type": "search",
-  "index": "tsdb-1d",
+  {%- if ingest_mode is defined and ingest_mode == "data_stream" %}
+  "index": "k8s",
+  {%- else %}
+  "index": "tsdb",
+  {% endif %}
```

Comment:
What is the point of this change?

Reply:
This is a mistake. I will revert this. This search is ran when running the `downsample` challenge and in that case the `tsdb-1d` index exists.